Public/script.js

```diff
@@ -63,11 +59,18 @@ async function makeTravelPlan() {
       }
   
       const data = await response.json();
-      document.getElementById("chatGPT-results").innerHTML =
-        data.choices[0].message.content;
+      const chatGPTResults = document.getElementById("chatGPT-results");
+      chatGPTResults.innerHTML = data.choices[0].message.content;
+      chatGPTResults.style.display = "block";
       console.log(response);
     } catch (error) {
       console.log("error fetching travel plan", error.message);
     }
   }
-  makeTravelPlan();
\ No newline at end of file
+  makeTravelPlan();
+
```

Comment:
I'll have a look into why this function keeps returning after pressing th ebutton.